### PR TITLE
update plugin to drone 1.0, golang 1.13 & go modules

### PR DESCRIPTION
**Pull request by techknowlogick**
also use drone manifests so that a single tag can support multi-arch

### CODE REVIEW

**Comment by techknowlogick** [comment]
@jonasfranz you may also want to connect this to cloud.drone.io so that you can use their agents.

File: .drone.yml [hunk 1]
```diff
@@ -1,109 +1,302 @@
...
+  image: plugins/docker:linux-amd64
+  settings:
+    dockerfile: docker/Dockerfile.linux.amd64
+    dry_run: true
+    password:
+      from_secret: docker_password
+    repo: jonasfranz/drone-crowdin
+    tags: linux-amd64
+    username:
+      from_secret: docker_username
```
Comment (jonasfranz) [inline .drone.yml]: Do we need the username, since `docker_username` is not available for pull requests?

File: .drone.yml [hunk 2]
```diff
@@ -1,109 +1,302 @@
...
+  - ./release/linux/amd64/drone-crowdin --help
+
+- name: dryrun
+  pull: always
+  image: plugins/docker:linux-amd64
+  settings:
+    dockerfile: docker/Dockerfile.linux.amd64
+    dry_run: true
+    password:
+      from_secret: docker_password
```
Comment (jonasfranz) [inline .drone.yml]: Do we need the username, since `docker_password` is not available for pull requests?

File: .drone.yml [hunk 3]
```diff
@@ -1,109 +1,302 @@
...
+  image: plugins/docker:linux-arm64
+  settings:
+    dockerfile: docker/Dockerfile.linux.arm64
+    dry_run: true
+    password:
+      from_secret: docker_password
+    repo: jonasfranz/drone-crowdin
+    tags: linux-arm64
+    username:
+      from_secret: docker_username
```
Comment (jonasfranz) [inline .drone.yml]: see above

File: .drone.yml [hunk 4]
```diff
@@ -1,109 +1,302 @@
...
+  image: plugins/docker:linux-arm
+  settings:
+    dockerfile: docker/Dockerfile.linux.arm
+    dry_run: true
+    password:
+      from_secret: docker_password
+    repo: jonasfranz/drone-crowdin
+    tags: linux-arm
+    username:
+      from_secret: docker_username
```
Comment (jonasfranz) [inline .drone.yml]: see above

File: .drone.yml [hunk 5]
```diff
@@ -1,109 +1,302 @@
...
+- name: manifest
+  pull: always
+  image: plugins/manifest:1
+  settings:
+    ignore_missing: true
+    password:
+      from_secret: docker_password
+    spec: docker/manifest.tmpl
+    username:
+      from_secret: docker_username
```
Comment (jonasfranz) [inline .drone.yml:276]: see above

**Comment by jonasfranz** [comment]
@techknowlogick Thank you for updating the plugin.

File: .drone.yml (hunk 5 above)
Comment (techknowlogick) [reply .drone.yml:276]: This is for manifest so it will be needed, but I will remove for the dry-run

**Comment by techknowlogick** [comment]
you're welcome. I've made the changes you requested.

**Comment by jonasfranz** [comment]
Did you change the docker repo by accident?

**Comment by jonasfranz** [comment]
@techknowlogick Did you change the docker repo by accident?

**Comment by techknowlogick** [comment]
@jonasfranz I did by accident. I had `jonasfranz/drone-crowdin` when I should put `jonasfranz/crowdin`. Let me put in another PR to fix. sorry.

**Pull request merged**